tinygrad/nn/__init__.py

```diff
@@ -48,16 +50,18 @@ class ConvTranspose2d:
   def __init__(self, in_channels, out_channels, kernel_size, stride=1, padding=0, output_padding=0, dilation=1, groups=1, bias=True):
     self.kernel_size = (kernel_size, kernel_size) if isinstance(kernel_size, int) else tuple(kernel_size)
     self.stride, self.padding, self.output_padding, self.dilation, self.groups = stride, padding, output_padding, dilation, groups
-    self.weight = Tensor.glorot_uniform(in_channels, out_channels//groups, *self.kernel_size)
-    self.bias = Tensor.zeros(out_channels) if bias else None
+    self.weight = Tensor.kaiming_uniform(in_channels, out_channels//groups, *self.kernel_size, a=math.sqrt(5))
+    bound = 1 / math.sqrt(prod(self.weight.shape[1:]))
+    self.bias = Tensor.uniform(out_channels, low=-bound, high=bound) if bias else None
 
   def __call__(self, x):
     return x.conv_transpose2d(self.weight, self.bias, padding=self.padding, output_padding=self.output_padding, stride=self.stride, dilation=self.dilation, groups=self.groups)
 
 class Linear:
-  def __init__(self, in_features, out_features, bias=True, initialization: str='kaiming_uniform'):
-    self.weight = getattr(Tensor, initialization)(out_features, in_features)
-    self.bias = Tensor.zeros(out_features) if bias else None
+  def __init__(self, in_features, out_features, bias=True):
+    self.weight = Tensor.kaiming_uniform(out_features, in_features, a=math.sqrt(5))
```

Comment:
I think you can just replace this with `Tensor.uniform` at least according to https://github.com/pytorch/pytorch/blob/683753fb0fc842afa9c43bc2a4576053af529429/torch/nn/modules/linear.py#L104-L111

Reply:
Meh, I'll leave it to match torch until there's tests. We should really add distribution tests for common torch layers, I did them manually in the CIFAR training.